src/context/common.ml

```diff
@@ -354,6 +354,7 @@ type context = {
 	mutable verbose : bool;
 	mutable foptimize : bool;
 	mutable platform : platform;
+	mutable custom_target : (string * string) option;
```

Comment:
Why do we need this? It made sense to me while everything was `Cross`, but now it seems like we can just use `com.platform` with `CustomTarget`.